Source/Serilog.Exceptions/Core/ExceptionEnricher.cs

```diff
@@ -20,22 +22,38 @@ public sealed class ExceptionEnricher : ILogEventEnricher
             new ReflectionTypeLoadExceptionDestructurer()
         };
 
+        public static readonly IExceptionPropertyFilter FilterIgnoringStackTraceAndTargetId =
```

Comment:
Reverse the naming again to `IgnoreStackTraceAndTargetIdExceptionFilter`

Reply:
Done.